src/main/java/uk/gov/companieshouse/web/pps/util/PenaltyUtilsService.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.companieshouse.web.pps.util;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import java.text.DecimalFormat;
+
+@Service
+public class PenaltyUtilsService {
+
+    public final String viewPenaltiesLateFilingReason;
```

Comment:
This property should be `private` visibility

Reply:
Fixed in latest commit